src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/constrainedtransfer/ConstrainedBoardingSearch.java

```diff
@@ -103,52 +111,102 @@ public boolean transferExist(int targetStopPos) {
     }
 
     /**
-     * Find the trip to board (trip index) and the transfer constraint
+     * Find the trip to board (trip index) and the transfer constraint.
+     * <p>
+     * This method set the following parameter if successful:
+     * <ul>
+     *     <li>{@code onTripIndex}
+     *     <li>{@code onTripIndex}
+     *     <li>{@code onTripIndex}
+     * </ul>
```

Comment:
I suppose on the list not all items are supposed to be onTripIndex?

Reply:
I fixed this.